extension/lib/mergeCartsOnLogin.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const ShopgateCartPipeline = require('./shopgate/CartExtensionPipeline')
+
+module.exports = async (context, input) => {
```

Comment:
JSDoc would be welcome. We'd usually put it in extension/typedefs.

Reply:
done